tests/neg/LiquidClass1.hs

```diff
@@ -0,0 +1,39 @@
+module LiquidClass where
+
+
+-- | Typing classes
+-- | Step 1: Refine type dictionaries:
+
+class Compare a where
+	cmax :: a -> a -> a
+	cmin :: a -> a -> a
+
+instance Compare Int where	
+{-@ instance Compare Int where 
+	cmax :: Odd -> Odd -> Odd ;
```

Comment:
I'm not sure, I think `div` and `mod` are used in a few tests, but I doubt we have an instance declared anywhere. Regardless, it parses properly :)